parm/atm/obs/lists/gdas_prototype_3d.yaml

```diff
@@ -3,11 +3,12 @@ observers:
 ##- !INC ${OBS_YAML_DIR}/sondes_prepbufr.yaml
 ##- !INC ${OBS_YAML_DIR}/atms_n20.yaml
 ##- !INC ${OBS_YAML_DIR}/aircraft.yaml
-- !INC ${OBS_YAML_DIR}/satwind_goes-16.yaml
```

Comment:
Makes sense to toggle off the observation type you are not testing (`satwind_goes-16.yaml`) and toggle on the observation type you are testing (`gnssro.yaml`).

As developers complete validation for their observation type(s), we should let `gdas_prototype_3d.yaml` in `feature/gdas-validation` reflect this by having an increasing list of active observation types.